suite-native/device-manager/src/components/PortfolioTrackerDeviceManagerContent.tsx

```diff
@@ -23,8 +24,18 @@ type NavigationProp = StackToStackCompositeNavigationProps<
     RootStackParamList
 >;
 
+const contentStyle = prepareNativeStyle(utils => ({
+    paddingHorizontal: utils.spacings.medium,
+
+    spacing: utils.spacings.medium,
+
+    paddingBottom: utils.spacings.medium,
+    marginTop: utils.spacings.medium,
```

Comment:
Why does this have `marginTop` but not `DeviceManagerContent` which is rendered in the same conditional?

Reply:
in DeviceManagerContent the children have it (DeviceList and WalletList)